examples/advanced-routing/src/MainLayout.css

```
@@ -0,0 +1,44 @@
+/* MainLayout styles */
+
+pelela[view-model="MainLayout"] {
```

Stylelint (17.14.0)

[error] Unknown type selector "pelela" (selector-type-no-unknown)

(selector-type-no-unknown)